web3.js/src/connection.ts

```diff
@@ -3085,9 +3116,66 @@ export class Connection {
           () => resolve({__type: TransactionStatus.TIMED_OUT, timeoutMs}),
           timeoutMs,
         );
+      } else if (Object.prototype.hasOwnProperty.call(strategy, 'nonceValue')) {
+        const INVALID_NONCE_ERROR_SIGIL = {} as const;
```

Comment:
What's a sigil?

Reply:
Oh oops, I meant to write ‘sentinel,’ and even then that nomenclature is a stretch. https://en.m.wikipedia.org/wiki/Sentinel_value

Reply:
Ah yeah sentinel is better